docs/notebooks/working_with_the_timeseries.ipynb

```diff
@@ -0,0 +1,112 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Working with the lsstseries `TimeSeries` object\n",
+    "\n",
+    "By contrast to the `Ensemble`, which operates on many lightcurves, the lsstseries `TimeSeries` object operates on a single lightcurve. \n",
+    "\n",
+    ">**Note**: This notebook is limited as `TimeSeries` has a very initial implementation"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## From the `Ensemble`\n",
+    "\n",
+    "A common use case for the `Timeseries` is pulling in an object of interest from the `Ensemble`. The `Ensemble` has a convenient exporter function for this purpose."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from lsstseries import Ensemble, TimeSeries\n",
+    "\n",
+    "ens = Ensemble()  # initialize an ensemble object\n",
+    "\n",
+    "# Read in data from a parquet file\n",
+    "ens.from_parquet(\"../../tests/lsstseries_tests/data/test_subset.parquet\",\n",
+    "                id_col='ps1_objid',\n",
+    "                time_col='midPointTai',\n",
+    "                flux_col='psFlux',\n",
+    "                err_col='psFluxErr',\n",
+    "                band_col='filterName')\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ts = ens.to_timeseries(88472935274829959) # provided a target object id\n",
+    "ts.data"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As a result, we get a multi-indexed `Pandas` DataFrame with data from a single lightcurve. The multi-index contains a band index as well as a integer index. We can operate on this as we normally would a `Pandas` DataFrame."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
```

Comment:
I noticed that matplotlib isn't listed as a requirement in pyproject.toml. But after I installed it, this worked as expected.

Reply:
Yeah, it's not needed at all for the code itself, but it is for if you want to run the notebooks locally. Not sure what the right call there is as to whether include it or not.

Reply:
One approach is to manually keep your docs/requirements.txt in sync with your pyproject.toml 'dev' dependencies section. You could also create a separate dependencies section in pyproject.toml, something like 'docs' that would just have dependencies needed to build notebooks that might not be necessary for other dev work. 

Or neither approach is fine too. As long as the docs/requirements.txt file is correct, then Sphinx/GitHub/readthedocs will be able to build and render the notebooks.